<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1824</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Harpoon</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pics/2332239.png</normaloff>pics/2332239.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="lvDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>256</width>
      <height>541</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>580</y>
      <width>165</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить изменения</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="settings">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBackup">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сделать бэкап</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>580</y>
      <width>177</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Восстановить из бэкапа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>580</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Восстановить исходный</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentConf">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>109</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Список файлов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>157</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Конфигурация файла:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>6</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Сохраненный бэкап</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сохраненный исходник</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="settings_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>30</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1824</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>Настройка директорий...</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Изменить директорию для хранения исходных файлов...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
